more_itertools/more.py

```diff
@@ -1590,7 +1590,7 @@ def groupby_transform(iterable, keyfunc=None, valuefunc=None):
     return ((k, map(valuefunc, g)) for k, g in res) if valuefunc else res
 
 
-def numeric_range(*args):
+class numeric_range:
```

Comment:
Would it be possible inherit from `collections.abc.Iterable` and `collections.abc.Sequence`? `range()` objects are marked as instances of each. Could add a test for this too.

Reply:
Good idea, but inherit from  ̀collections.abc.Iterable` is unnecessary since a `Sequence` is already an `Iterable`. I will mark `numeric_range` as  ̀collections.abc.Sequence` and  ̀collections.abc.Hashable`.

Reply:
Done.